travis/publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "********************************"
+echo "***** Publishing Artifacts *****"
+echo "********************************"
+echo "Branch: ${TRAVIS_BRANCH}"
+echo "Tag: ${TRAVIS_TAG}"
+echo "JDK Version: ${TRAVIS_JDK_VERSION}"
+
+#Weird comparison comparing branch to tag because when you 'git push --tags'
+#the branch somehow becomes the tag value
+#github issue: https://github.com/travis-ci/travis-ci/issues/1675
+test "${TRAVIS_BRANCH}" = "${TRAVIS_TAG}" &&
```

Comment:
maybe use something more like the if statement here, as it emits what's going on? https://github.com/Netflix/feign/blob/master/installViaTravis.sh